client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -10,9 +10,8 @@ import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
-import getSiteStatsQueryDate from 'state/selectors/get-site-stats-query-date';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { isRequestingSiteStatsForQuery } from 'state/stats/lists/selectors';
+import { isRequestingSiteStatsForQuery, getSiteStatsQueryDate } from 'state/stats/lists/selectors';
```

Comment:
If you favor an uber nit here: I tend to alphabetize not only imports on a per-line basis, but also the multiple imports from the same module. Wanna do that? 😉

Reply:
Done! Assuming you order by import path 😄